sample.env

```diff
@@ -8,9 +8,3 @@ CSRF_TRUSTED_ORIGINS=localhost:3000,localhost:8000,localhost:8080,test.example.c
 SSL_PROTECTION=False
 AUTH_STAFF_EMAIL_DOMAINS=mozillafoundation.org
 LOGIN_ALLOWED_REDIRECT_DOMAINS=test.example.com:3000
-
```

Comment:
change history of this file: https://github.com/mozilla/network-pulse-api/commits/master/sample.env